InnoGotchiFrontend/InnoGotchi.BLL/DTO/PaginatedListDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace InnoGotchi.BLL.DTO
+{
+    public class PaginatedListDTO<T> where T : class
+    {
+        public int PageIndex { get; set; }
+        public int TotalPages { get; set; }
+        public List<T>? Items { get; set; } = null;
```

Comment:
I am not really understanding your point. Should I rename this class? And what I need to do with Items?

Reply:
In some case we don't really need List as collection. If I want to pull array, or IEnumerable - I need to have such option. So if you will work not with List of T objects, but with T object, it will be more useful. In this case I can use any collection, that I really want.